internal/ui/ui_js.go

```diff
@@ -717,6 +718,27 @@ func (u *userInterfaceImpl) Window() Window {
 	return &nullWindow{}
 }
 
+type Monitor struct{}
+
+func (m *Monitor) Bounds() image.Rectangle {
+	screen := windows.Get("screen")
+	w := screen.Get("width").Int()
+	h := screen.Get("height").Int()
+	return image.Rect(0, 0, w, h)
+}
+
+func (m *Monitor) Name() string {
+	return ""
+}
+
+func (u *userInterfaceImpl) AppendMonitors(mons []*Monitor) []*Monitor {
+	return nil
+}
+
+func (u *userInterfaceImpl) Monitor() *Monitor {
+	return nil
```

Comment:
So we plan to return a dummy monitor here, right?

Reply:
Correct. It should be the same monitor as returned by AppendMonitors